contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -868,11 +894,19 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
         nonReentrant
     {
         // to add liquidity to the clPool
-        IERC20(WETH).safeApprove(address(positionManager), type(uint256).max);
+        IERC20(OETHb).safeApprove(address(positionManager), 0);
         IERC20(OETHb).safeApprove(address(positionManager), type(uint256).max);
         // to be able to rebalance using the swapRouter
-        IERC20(WETH).safeApprove(address(swapRouter), type(uint256).max);
+        IERC20(OETHb).safeApprove(address(swapRouter), 0);
```

Comment:
nit: Any reason to use safeApprove for OETHb? It behaves as we expect it to with just `approve` as well

Reply:
nice catch, correcting